sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs

```diff
@@ -795,41 +799,89 @@ public async Task<AzureOperationResponse<SignalRKeys>> ListKeysWithHttpMessagesA
         /// this value from the Azure Resource Manager API or the portal.
         /// </param>
         /// <param name='resourceName'>
-        /// The name of the SignalR resource.
+        /// The name of the resource.
         /// </param>
-        /// <param name='parameters'>
-        /// Parameter that describes the Regenerate Key Operation.
+        /// <param name='customHeaders'>
+        /// The headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        public async Task<AzureOperationResponse<SignalRResource>> CreateOrUpdateWithHttpMessagesAsync(SignalRResource parameters, string resourceGroupName, string resourceName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            // Send Request
+            AzureOperationResponse<SignalRResource> _response = await BeginCreateOrUpdateWithHttpMessagesAsync(parameters, resourceGroupName, resourceName, customHeaders, cancellationToken).ConfigureAwait(false);
+            return await Client.GetPutOrPatchOperationResultAsync(_response, customHeaders, cancellationToken).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Operation to delete a resource.
+        /// </summary>
+        /// <param name='resourceGroupName'>
+        /// The name of the resource group that contains the resource. You can obtain
+        /// this value from the Azure Resource Manager API or the portal.
+        /// </param>
+        /// <param name='resourceName'>
+        /// The name of the resource.
         /// </param>
         /// <param name='customHeaders'>
         /// The headers that will be added to request.
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        public async Task<AzureOperationResponse<SignalRKeys>> RegenerateKeyWithHttpMessagesAsync(string resourceGroupName, string resourceName, RegenerateKeyParameters parameters = default(RegenerateKeyParameters), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse> DeleteWithHttpMessagesAsync(string resourceGroupName, string resourceName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
         {
             // Send request
-            AzureOperationResponse<SignalRKeys> _response = await BeginRegenerateKeyWithHttpMessagesAsync(resourceGroupName, resourceName, parameters, customHeaders, cancellationToken).ConfigureAwait(false);
+            AzureOperationResponse _response = await BeginDeleteWithHttpMessagesAsync(resourceGroupName, resourceName, customHeaders, cancellationToken).ConfigureAwait(false);
```

Comment:
did this operation become lro?

Reply:
sorry，whats lro stand for?

Reply:
long-running operation.  It looks like this is a bad diff.